rom/dev/tests/test_fmcalias_derivation.rs

```diff
@@ -224,6 +225,95 @@ fn test_pcr_log() {
     );
 }
 
+#[test]
+fn test_pcr_log_no_owner_key_fuse() {
```

Comment:
Can we rename this to test_pcr_log_no_owner_key_digtest_fuse?

Reply:
Done